packages/zmarkdown/server/handlers.js

```diff
@@ -147,7 +149,11 @@ module.exports = function markdownHandlers (Raven) {
         }
       }
 
-      processors[key] = zmarkdown({remarkConfig: remark, rebberConfig: rebber}, target)
+      processors[key] = zmarkdown({remarkConfig: remark,
```

Comment:
Please move the first key on the next line, with the same indentation as the other keys

Reply:
:O I really wrote dirty code yesterday on the train…